### DISCUSSION TITLE
Typo on purchase success setup page

### DISCUSSION THREAD

**Discussion by nick-w-nick** [community/community#19064] [Copilot Conversations] [answered]
I wasn't sure where exactly to post this, but I noticed a minor typo (see image) on the confirmation/setup page that you are redirected to after purchasing Copilot (https://github.com/github-copilot/signup/settings).

![alredy-typo](https://user-images.githubusercontent.com/43578531/174859415-11f651de-33bd-4702-b915-a8b02f0946dc.png)

Let me know if this issue would be better handled another way/via support.

Thanks!

**Comment by davecheney** [comment, marked as answer]
Thank you, it's already fixed, just waiting in the deployment pipeline.
